### PR TITLE
Update about and recruitment sections

### PR DESCRIPTION
**Pull request by bfbachmann**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,9 +94,8 @@ <h2>DRIVER</h2>
     <div id="about">
       <h1>About Us</h1>
       <p class="large">
-        Launch Pad is UBC's student-run software engineering team.<br />
-        We aim to be the leading organization on campus for students to build their
-        software development skills in a collaborative and professional environment.
+          UBC Launch Pad is a student-run software engineering team devoted to building software projects in a collaborative and professional environment.<br />
+          Our primary objective is to be the best space at UBC for students to meet other tech enthusiasts, build projects in a team setting, gain experience with professional development tools and principles, and share their knowledge with others. We form teams based on common interests and project ideas of our own choosing, and hack on our projects to completion! We are also lucky enough to have sponsors which help provide our teams with necessary resources, like servers, to help us achieve our goals.
```
Comment (jordanschalm) [inline index.html]: Life is easier when you keep line length under ~80

**Review by jordanschalm — APPROVED** [review]

**Comment by bobheadxi** [comment]
If this is for #39 I'm not sure it really addresses comments from the (initially heavily downvoted) [reddit post](https://www.reddit.com/r/UBC/comments/7n6jnq/ubc_launch_pad_is_recruiting_developers_and/):

> They are just trying to make this club an elite club... Simply put, not worth the effort

and this one, which is ended up being the top voted comment:

> ... I would say the application process is quite competitive and isn't worth the effort. A club like Code the Change has a far greater breadth of projects and is equally effective in terms of personal projects for resumes.

and concerns about how welcoming the club is:

> Is LP still going to be for students of any level going forwards? I got asked a time complexity question when the only courses I'd taken were 110 and 121. First technical interview I'd ever done too...

I think replies like this one helped (emphasis mine, I'm guessing @jordanschalm wrote this?):

> **We want the club to be useful for members**, so we want people to be able to build projects and learn new things. After doing this for a few years, we’ve found that building projects at a reasonable pace requires one or two more experienced members per team who can lead the way and act as a resource for folks who may not as much experience.

I wonder if this place might be a good place to put some emphasis on this 🤷‍♂️

**Comment by jordanschalm** [comment]
Agree with @bobheadxi we need a FAQ separate from this (I think this was your plan anyway @bfbachmann)

**Comment by bfbachmann** [comment]
Yea I was planning on adding a FAQ section later.

File: index.html (hunk 1 above)
Comment (bfbachmann) [reply index.html]: Good point. I wrote this as though it were Markdown 😬

**Pull request merged**